tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1098,6 +1119,25 @@ def _serialize_and_track_engine(node):
     signatures = {
         key: value for key, value in self._saved_model.signatures.items()
     }
+
+    # Set allow_build_at_runtime=False if asked by user.
+    # This attribute is set here because build() needs it to be True
+    # in order to build engines.
+    if not self._conversion_params.allow_build_at_runtime:
+      def _reset_allow_build_at_runtime(node):
+        node.attr["allow_build_at_runtime"].b = False
```

Comment:
@aaroey shouldn't this be `_allow_build_at_runtime`?
I wonder why tests don't fail.